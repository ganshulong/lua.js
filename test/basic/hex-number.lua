print(0x12345678)
print(0x3.4)