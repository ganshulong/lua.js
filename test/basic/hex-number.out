305419896
3.25
